src/components/Button/InputButton.tsx

```diff
@@ -0,0 +1,52 @@
+import classNames from "classnames";
+import PropTypes from "prop-types";
+import React from "react";
+
+import "lbh-frontend/lbh/components/lbh-button/_button.scss";
+import "govuk-frontend/govuk/components/button/button.js";
+
+/**
+ * The proptypes for {@link InputButton}
+ */
+export interface InputButtonProps {
+  id?: string;
+  className?: string;
+  isStartButton?: boolean;
+  name?: string;
+  type?: React.ButtonHTMLAttributes<HTMLButtonElement>["type"];
```

Comment:
Is this really optional? Without the type, this will be a regular text input, not a button. Maybe it should default to `"button"`?